src/routing/approved.rs

```diff
@@ -746,6 +858,7 @@ impl Approved {
         let src_name = match src {
             SrcLocation::Node(name) => name,
             SrcLocation::Section(prefix) => prefix.name(),
+            _ => unimplemented!(),
```

Comment:
same as previous. try not to use wild card match and `unimplemented`

Reply:
Yep :+1:, still wip code, so this is the unimplemented part of the work in progress.